optaplanner-webexamples/pom.xml

```diff
@@ -25,6 +25,7 @@
   <properties>
     <version.org.wildfly>10.1.0.Final</version.org.wildfly>
   </properties>
+
   <build>
```

Comment:
@psiroky Was this intended for formatting consistency? Please confirm that the webexamples module doesn't need the `java.module.name` property.

Reply:
Probably not intended. The property is not needed uses packaging `war`. We only want module names for jars.